node-tests/acceptance/exam-test.js

```diff
@@ -129,4 +129,13 @@ describe('Acceptance | Exam Command', function() {
       });
     });
   });
+
+  describe('Weighted', function() {
+    it('runs tests by order of test type', function(done) {
+      exec('ember exam --weighted --path acceptance-dist', function(_, stdout) {
```

Comment:
besides asserting all test ran, can we also assert the order of the tests as well?
Although I see we have unit tests for this, since this is an acceptance tests that tests the full flow, i think it will be beneficial to have that assert as well.

Reply:
Agree with @step2yeung on this one.